### PR TITLE
Easier to deploy by avoiding merge conflicts in production

### PR DESCRIPTION
**Pull request by SavinaRoja**
https://github.com/mouse-reeve/bookwyrm/issues/857

To address the points in https://github.com/mouse-reeve/bookwyrm/issues/857#issuecomment-812949001

### CODE REVIEW

File: .env.dev.example [hunk 1]
```diff
@@ -26,13 +27,23 @@ POSTGRES_HOST=db
...
+#REDIS_ACTIVITY_PASSWORD=redispassword345
 
-# Celery config with redis broker
+# Redis as celery broker
+#REDIS_BROKER_PORT=6379
+#REDIS_BROKER_PASSWORD=redispassword123
 CELERY_BROKER=redis://redis_broker:6379/0
 CELERY_RESULT_BACKEND=redis://redis_broker:6379/0
 
+#FLOWER_PORT=8888
```
Comment (mouse-reeve) [inline .env.dev.example:38]: Flower is used in both dev and prod, so the port here should be uncommented I think

**Review by mouse-reeve — CHANGES_REQUESTED** [review]

File: docker-compose.yml [hunk 2]
```diff
@@ -20,6 +20,8 @@ services:
       - pgdata:/var/lib/postgresql/data
     networks:
       - main
+    ports:
```
Comment (SavinaRoja) [inline docker-compose.yml:23]: I found this helpful during development, being able to connect directly with a database tool like DBeaver.

**Comment by SavinaRoja** [comment]
I believe that a change will be required in the production branch after this. Changing https://github.com/mouse-reeve/bookwyrm/blob/c50edc9d3fbd15994c241c9f3b0a55efc11eb918/docker-compose.yml#L21 to `command: bash ./certbot.sh` and removing the following line.

File: .env.dev.example (hunk 1 above)
Comment (SavinaRoja) [reply .env.dev.example]: This should be fixed now.

**Review by mouse-reeve — APPROVED** [review]

**Comment by mouse-reeve** [comment]
Sorry for taking so long to get back to you on this! I think it's good to go pending the merge conflicts (and let me know if I can help resolving them)

**Comment by SavinaRoja** [comment]
Resolved the merge conflicts. I expect merging this this will introduce conflicts with https://github.com/mouse-reeve/bookwyrm/pull/925

**Pull request merged**